src/test/java/Tests/TestWithUsingComponent.java

```diff
@@ -0,0 +1,23 @@
+package Tests;
+
+import BaseTest.TestRunner;
+import Components.Buttons.BuyButton;
+import Components.Frameworks.ProductFrame;
+import Components.Frameworks.ProductsFrame;
+import Components.Utils.Component;
+import Steps.ProductSteps;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestWithUsingComponent extends TestRunner {
```

Comment:
`TestUsingComponents` or `TestWithComponents`. No reason to have both words in class name